docs/user/ppl/functions/ip.rst

```diff
@@ -20,19 +20,19 @@ Argument type: STRING, STRING
 
 Return type: BOOLEAN
 
-Example:
+Example::
 
-    os> source=weblogs | where cidrmatch(host, '199.120.110.0/24') | fields host
-    fetched rows / total rows = 1/1
-    +----------------+
-    | host           |
-    |----------------|
-    | 199.120.110.21 |
-    +----------------+
+    > source=weblogs | where cidrmatch(host, '1.2.3.0/24') | fields host, url
+    fetched rows / total rows = 2/2
+    +---------+--------------------+
+    | host    | url                |
+    |---------|--------------------|
+    | 1.2.3.4 | /history/voyager1/ |
+    | 1.2.3.5 | /history/voyager2/ |
+    +---------+--------------------+
 
 Note:
- - `ip` can be an IPv4 or an IPv6 address
- - `cidr` can be an IPv4 or an IPv6 block
- - `ip` and `cidr` must be either both IPv4 or both IPv6
- - `ip` and `cidr` must both be valid and non-empty/non-null
+ - `ip` can be an IPv4 or IPv6 address
+ - `cidr` can be an IPv4 or IPv6 block
+ - `ip` and `cidr` must both be valid and non-missing/non-null
```

Comment:
Slight update to the documentation here. Specifically, removed "`ip` and `cidr` must be either both IPv4 or both IPv6", which isn't true (this function works fine on SQL even if the address and block use different IP versions); and changes "non-empty" to "non-missing", which seems to be the more common terminology within the SQL project.